packages/idyll-template-projects/README.md

```diff
@@ -0,0 +1,4 @@
+# idyll-layouts
+Layouts that ship by default in new idyll projects.
+
+*See http://idyll-lang.org for complete documentation.*
```

Comment:
Should be updated to reflect the `idyll-template-projects` purpose. This can probably be borrowed from https://github.com/idyll-lang/generator-idyll

Reply:
Oops, missed this too. Updated it with some usage info